frontend/src/component/executiveDashboard/componentsStat/FlagStats/FlagStats.tsx

```
@@ -102,7 +106,7 @@ export const FlagStats: React.FC<IFlagStatsProps> = ({
                             </Typography>
                         </StyledTextContainer>
                         <StyledFlagCountPerUser>
-                            {flagsPerUser}
+                            {flagsPerUser ?? 'N/A'}
```

So the only time this'll evaluate to `N/A` is if flagsPerUser is `null`, right? Is that a value that we expect some times?

How is flags per user calculated? When will this be null? If there are no flags, we can just say it's 0, can't we?

---

If `flagsPerUser` has been changed to always being a string now, can we remove the fallback? It will never happen, so leaving it in is a little misleading, I think.
```suggestion
                            {flagsPerUser}
```